presto-pinot/presto-pinot-lib/src/main/java/com/facebook/presto/pinot/PinotConfig.java

```diff
@@ -0,0 +1,400 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.pinot;
+
+import com.facebook.airlift.configuration.Config;
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.units.Duration;
+import io.airlift.units.MinDuration;
+
+import javax.annotation.Nullable;
+import javax.validation.constraints.NotNull;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public class PinotConfig
+{
+    public static final int DEFAULT_LIMIT_LARGE_FOR_SEGMENT = Integer.MAX_VALUE;
```

Comment:
Make it private and explicit  write the value in the test, so that any change in default value is covered by test.